src/types/QIX.ts

```diff
@@ -39,9 +39,49 @@ export interface RowHeight {
   linesCount: number;
 }
 
+export interface PaletteColor {
+  index: number;
+  color: string | null;
+}
 export interface Component {
   key: "general" | "theme";
   rowHeight?: RowHeight;
+  header?: {
+    fontFamily?: string;
+    fontSize?: number;
+    fontColor?: PaletteColor;
+  };
+  content?: {
+    fontFamily?: string;
+    fontSize?: number;
+    fontColor?: PaletteColor;
+  };
+}
+
+export interface CurrentTheme {
```

Comment:
Not sure if this fits into `QIX` types. But lets keep it here for now and later figure out with it means to be a qix type